src/lib.rs

```diff
@@ -339,3 +340,13 @@ pub mod widgets;
 pub use self::terminal::{CompletedFrame, Frame, Terminal, TerminalOptions, Viewport};
 
 pub mod prelude;
+
+/// re-export the `crossterm` crate so that users don't have to add it as a dependency
```

Comment:
Maybe reword this like the following for making it more user facing rather than a 3rd person comment:

> Re-export of the [crossterm] crate

And also making this a link would be useful too

Reply:
It's not avoiding the import, it's simplifying the Cargo.toml setup as the version reexported will always be the correct one. Not sure how to write that in a relatively short way.

And yeah, a link is pretty simple to add here, but on the other hand its just below… so both variants seem fine to me.

Reply:
Neither docs.rs or rust-analyzer show this comment.
- docs.rs shows `crossterm` in the re-exports section, without a comment
- hovering over `ratatui::crossterm` shows the docs for crossterm, not this comment

There is a `#[doc(inline)]` attribute that can be applied, which moves re-exports into where the item would be if it were part of the current module, and uses this comment. E.g. this would move to the modules section, but that's not really appropriate as `crossterm` isn't a submodule of Ratatui, it's much more useful to call it out as what it is - a re-export.

You could also ask why this is a `///` if it doesn't appear in rustdoc, but that seems appropriate here because `///` is generally seen as the right comment to write against an item (rather than `//`).